src/AspNet.Security.OAuth.AdobeIO/AspNet.Security.OAuth.AdobeIO.csproj

```
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
```

Could you rebase and then change this to match a change I made earlier today please? (c4d9870de15e71effd3f1f8b3b0188b500bf08f1)

```suggestion
    <TargetFrameworks>$(DefaultNetCoreTargetFramework)</TargetFrameworks>
```

---

Done